utility/serial.cpp

```diff
@@ -92,6 +92,60 @@ void ArdusatSerial::begin(unsigned long baud, bool setXbeeSpeed)
   }
 }
 
+/**
+ * Begin serial communications with a Sparkfun BlueSMiRF module at the specified baud
+ * rate.
+ *
+ * Note that baud rates above ~57600 are not well-supported by SoftwareSerial, and 
+ * even 57600 may cause some bugs.
+ *
+ * @param baud rate for serial communications
+ */
+void ArdusatSerial::beginBluetooth(unsigned long baud)
+{
+  if (_mode == SERIAL_MODE_HARDWARE || _mode == SERIAL_MODE_HARDWARE_AND_SOFTWARE) {
+    Serial.begin(baud);
+
+    _soft_serial->end();
+    _soft_serial->begin(115200); // bluesmirf defaults to baudrate of 115200
```

Comment:
There is a set baudrate command that persists. But this is the best way I found that still allows the user to switch the baudrate in a similar fashion to `Serial.begin(xxxx);` from the Arduino's `setup()` function.